build.gradle

```diff
@@ -52,6 +56,7 @@ buildscript {
 
   dependencies {
     classpath deps.kotlin.gradlePlugin
+    classpath deps.kotlinNative.gradlePlugin
```

Comment:
nit: this might look a bit cleaner as `deps.kotlin.native.gradlePlugin`

Reply:
Yeah, that is much better.